examples/ui-demo/src/app/providers.tsx

```diff
@@ -1,29 +1,45 @@
 "use client";
 
 import { ToastProvider } from "@/contexts/ToastProvider";
+import { convertDemoConfigToUiConfig } from "@/state/store";
 import { AlchemyClientState } from "@account-kit/core";
+import { AlchemyAccountProvider } from "@account-kit/react";
 import { QueryClientProvider } from "@tanstack/react-query";
 import { PropsWithChildren, Suspense } from "react";
 import { ConfigContextProvider, ConfigSync } from "../state";
-import { Config, queryClient } from "./config";
+import { alchemyConfig, Config, queryClient } from "./config";
 
 export const Providers = (
   props: PropsWithChildren<{
     initialState?: AlchemyClientState;
     initialConfig?: Config;
   }>
 ) => {
+  const localAlchemyConfig = {
+    ...alchemyConfig,
+    ui: props.initialConfig
+      ? convertDemoConfigToUiConfig(props.initialConfig)
+      : alchemyConfig.ui,
+  };
```

Comment:
ensures that the initial config is inline with the cached config if there is one